src/modules/service-catalog/components/useItemFormFields.tsx

```diff
@@ -71,6 +71,10 @@ const fetchTicketFields = async (
   const formData = await formResponse.json();
   const fieldsData = await fieldsResponse.json();
 
+  if (!formData.ticket_form.active) {
+    throw new Error("Error fetching form data");
```

Comment:
Can we be more specific in the error here? Would help when debugging the issue cos we'd see it in the console right away.

Similar to what we do above with "Associated lookup field not found", e could say "Associated ticket form is not active".